web/src/styles/globals.scss

```diff
@@ -1,33 +1,17 @@
-@import "./components.scss";
 @import "./text.scss";
 @import "./btns.scss";
 @tailwind base;
-@tailwind components;
 @tailwind utilities;
 
 :root {
-	--foreground-rgb: 0, 0, 0;
-	--background-start-rgb: 214, 219, 220;
-	--background-end-rgb: 255, 255, 255;
-	--color-primary: #79fb2a;
-	--color-secondary: #45ff02;
-	--color-tertiary: #e74c3c;
-	--c-gray-1: #313131;
-	--c-gray-2: #222222;
-	--c-gray-3: #1b1b1b;
-	--primary-text: #333;
-	--secondary-text: #555;
+	--c-primary: #45ff02;
+	--c-gray-1: rgba(199, 199, 198, 25);
+	--c-gray-2: rgba(127, 127, 127, 20);
```

Comment:
I'd prefer leaving it as hex code. There is also the possibility to add opacity with hex codes. You just have to append two digits. 

Refer to https://www.w3schools.com/cssref/tryit.php?filename=trycss_color_hexa and https://gist.github.com/lopspower/03fb1cc0ac9f32ef38f4